language/dsl/python/src/flow.py

```diff
@@ -0,0 +1,122 @@
+
+
+from typing import Any, Dict, Optional, List
+from navigation import Navigation, NavigationFlowEndState
+from schema import Schema
+from view import View
+
+
+DataModel = Dict[Any, Any]
+
+
+class FlowResult:
```

Comment:
W/ XLR Python generation in the works, how close are we to being able to generate these types? I know at least we would need to generate XLRs on Player core types.